Makefile

```diff
@@ -18,11 +18,15 @@ $(VENV)/bin/pytest: $(VENV)/bin/python requirements.txt
 	touch $@
 
 $(VENV)/bin/sphinx-build: $(VENV)/bin/pip
-	$(VENV)/bin/pip install sphinx
+	$(VENV)/bin/pip install sphinx sphinx-autobuild
+	$(VENV)/bin/pip install Flask-Sphinx-Themes
```

Comment:
This has been moved to the "docs" tox env.
I think a new env "docs-dev" would be good then, where sphinx-autobuild gets used then (sounds interesting, have to check it out myself).

Reply:
Or "live-docs" - according to the Makefile target you have added, and which could still be in here, but would use tox then also.

Reply:
That's another part of your comment that I don't seem to understand, do you need me to perform any updates on this code?

Reply:
You will run into a conflict when rebasing.
Handling docs has moved from the Makefile to tox.ini (and the Makefile just calls it).
Therefore the `sphinx-autobuild` dep needs to be installed there, and should become a new tox env probably (`live-docs`).
Feel free to leave this out though, of course.

Reply:
I added it as a convenience. 

I don't see any point of delaying documentation changes (which is the whole idea of this PR) in order to discuss ``live-docs`` and sphinx-autobuild.

Please ignore changes and other non-essential stuff.